app/models/manageiq/providers/ibm_power_vc/cloud_manager.rb

```diff
@@ -36,6 +36,10 @@ def self.vm_vendor
     "ibm_power_vc"
   end
 
+  def console_url
+    "https://" + self.endpoint.hostname
```

Comment:
```suggestion
    "https://#{endpoint.hostname}"
```

gets rid of the string and just chucks the string in there

thnx for the pr